resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -5,11 +5,12 @@ pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
 
 restartfile=<%= EZBake::Config[:restart_file] %>
 start_timeout=<%= EZBake::Config[:start_timeout] %>
-stop_timeout=60
 dir=$(dirname "$restartfile")
 realname="<%= EZBake::Config[:real_name] %>"
 PIDFILE="/var/run/puppetlabs/${realname}/${realname}.pid"
 
+[ -e "${INSTALL_DIR}/ezbake-functions.sh" ] && . "${INSTALL_DIR}/ezbake-functions.sh"
```

Comment:
@haus - you have any strong opinion as to whether we check for the existence of the file or not?  If we do want to check for it, better to wrap a `set -x` and `set +x` around it or echo a specific message and error out.  I'm kind of leaning toward the latter, I guess, but don't care too much.